server/controllers/Conversations.js

```diff
@@ -128,6 +137,7 @@ module.exports.leaveConversation = async (req, res) => {
 
     // Validate request
     if (!userId|| !conversationId) {
+        logger.error("missing " + (userId ? "conversation" : "user") + " id");
```

Comment:
remove userId and user make it: "missing conversationId"

Reply:
Or "no conversation selected"?

Reply:
Alot of these error / info messages (the ones being send back with http res) are not necessarily appropriate to display to the user and in some case the error msg / info message we would want to send back to the user may not make sense to have the same message being logged on the server. For example:

For example, if the admin tries to leave the convo without choosing a successor, it makes sense that the current error message sent back to the client is "You're the only admin. You must choose a successor." but to put a message with "You" in the server logs doesn't make as much sense (my opinion).

Reply:
Yh. while rereading I noticed that. Some other places would be where the message has identifiers for the created or deleted object? Maybe just say that an object was created or deleted, don't log the actual id or displayName? Make the logged messages generic?

Reply:
Not sure though. It could be useful to log the identifiers?

Reply:
If the logged messages are generic, it may be harder to debug using the logs and that's one of the reasons for the log files in the first place so.

Reply:
Ok got it. Then do we want to log who is performing the request? `User (${userId}) successfully created conversation (${conversationId})`,  and X error happened when user (${userId}) tried to do Y, etc. It will be in the url but rather than searching through the url it could just be in the message. Not necessary, but could be useful.

Reply:
Actually yes, have the id in the message because there is no guarantee that the error would be logged directly after the url was logged for that particular user

Reply:
Ok great.